---
title: AsposeBadServerResponceException
second_title: Referencia de la API de Aspose.Email para .NET
description: Inicializa una nueva instancia delAsposeBadServerResponceExceptionaspose.email/asposebadserverresponceexception clase con un mensaje de error especificado.
type: docs
weight: 10
url: /es/net/aspose.email/asposebadserverresponceexception/asposebadserverresponceexception/
---
## AsposeBadServerResponceException(string) {#constructor_2}

Inicializa una nueva instancia del[`AsposeBadServerResponceException`](../../asposebadserverresponceexception) clase con un mensaje de error especificado.

```csharp
public AsposeBadServerResponceException(string message)
```

| Parámetro | Escribe | Descripción |
| --- | --- | --- |
| message | String | Un mensaje que describe el error. |

### Ver también

* class [AsposeBadServerResponceException](../../asposebadserverresponceexception)
* espacio de nombres [Aspose.Email](../../asposebadserverresponceexception)
* asamblea [Aspose.Email](../../../)

---

## AsposeBadServerResponceException() {#constructor}

Inicializa una nueva instancia del[`AsposeBadServerResponceException`](../../asposebadserverresponceexception) clase.

```csharp
public AsposeBadServerResponceException()
```

### Ver también

* class [AsposeBadServerResponceException](../../asposebadserverresponceexception)
* espacio de nombres [Aspose.Email](../../asposebadserverresponceexception)
* asamblea [Aspose.Email](../../../)

---

## AsposeBadServerResponceException(string, Exception) {#constructor_3}

Inicializa una nueva instancia del[`AsposeBadServerResponceException`](../../asposebadserverresponceexception) clase con un mensaje de error especificado y una referencia a la excepción interna que es la causa de esta excepción.

```csharp
public AsposeBadServerResponceException(string message, Exception innerException)
```

| Parámetro | Escribe | Descripción |
| --- | --- | --- |
| message | String | El mensaje de error que explica el motivo de la excepción. |
| innerException | Exception | La excepción que es la causa de la excepción actual, o una referencia nula (Nada en Visual Basic) si no se especifica ninguna excepción interna. |

### Ver también

* class [AsposeBadServerResponceException](../../asposebadserverresponceexception)
* espacio de nombres [Aspose.Email](../../asposebadserverresponceexception)
* asamblea [Aspose.Email](../../../)

---

## AsposeBadServerResponceException(Exception) {#constructor_1}

Inicializa una nueva instancia del[`AsposeBadServerResponceException`](../../asposebadserverresponceexception) clase con una referencia a la excepción interna que es la causa de esta excepción.

```csharp
public AsposeBadServerResponceException(Exception innerException)
```

| Parámetro | Escribe | Descripción |
| --- | --- | --- |
| innerException | Exception | La excepción que es la causa de la excepción actual, o una referencia nula (Nada en Visual Basic) si no se especifica ninguna excepción interna. |

### Ver también

* class [AsposeBadServerResponceException](../../asposebadserverresponceexception)
* espacio de nombres [Aspose.Email](../../asposebadserverresponceexception)
* asamblea [Aspose.Email](../../../)

<!-- DO NOT EDIT: generated by xmldocmd for Aspose.Email.dll -->
